easylist_adult/adult_specific_block.txt

```diff
@@ -411,6 +411,7 @@
 ||jav-porn.net/js/popup.js
 ||javfor.me^*/banner/
 ||javhub.net/img/r.jpg
+||javporn.best/asset/bann.js
 ||javporn.in/clicunder.js
```

Comment:
This domain is hijacked by `parkingcrew.net.` and no longer serving adult material by google ads

```
javporn.in.     3600    IN      NS      ns2.parkingcrew.net.
javporn.in.     3600    IN      NS      ns1.parkingcrew.net.
```

<details><summary>Own note</summary>

https://www.mypdns.org/T635

</details>

Reply:
Hmm...I get  "Error 1020 Access denied".

Reply:
Yeah, parkingsuckers currently luckily having issues at the moment :smiley: :joy: 

The same happens with one if the other domain creepers but in reality [parkingcrew](https://www.mypdns.org/T635) as well

```shell
javstreaming.net.       172800  IN      NS      ns1.hastydns.com.
javstreaming.net.       172800  IN      NS      ns2.hastydns.com.
```

Reply:
Thx, I removed the rule from my list. Pingging @ryanbr in case not aware of.